quanta-t41/config.json

```diff
@@ -35,7 +35,6 @@
     },
     "workflowRoot": "workflows",
     "taskRoot": "tasks",
-    "httpTemplateRoot": "templates",
-    "httpStaticRoot": "static"
```

Comment:
This is a required element that should not be removed and the user is required to set it up as per the quanta-t41/README.md